Streetcode/Streetcode.XUnitTest/MediatRTests/AdditionalContent/AdditionalContentTests/TagTests/GetAllTagsRequestHandlerTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Xunit;
+using Moq;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore.Query;
+using Streetcode.DAL.Entities.AdditionalContent;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System.Linq.Expressions;
+using Streetcode.BLL.DTO.AdditionalContent;
+using Streetcode.BLL.MediatR.AdditionalContent.Tag.GetAll;
+
+namespace Streetcode.XUnitTest.MediatRTests.AdditionalContent.TagTests
+{
+    public class GetAllTagsRequestHandlerTests
+    {
+        private readonly Mock<IRepositoryWrapper> _mockRepo;
+        private readonly Mock<IMapper> _mockMapper;
+
+        public GetAllTagsRequestHandlerTests()
+        {
+            _mockRepo = new Mock<IRepositoryWrapper>();
+            _mockMapper = new Mock<IMapper>(); 
+        }
+
+        private readonly List<Tag> tags = new List<Tag>()
+        {
+            new Tag
+            {
+                Id = 1,
+                Title = "some title 1"
+            },
+            new Tag
+            {
+                Id = 2,
+                Title = "some title 2"
```

Comment:
use variable or inline data for them. also avoid using magic strings (inline data is an option)